src/main/java/htsjdk/tribble/annotation/Strand.java

```diff
@@ -24,29 +24,102 @@
 package htsjdk.tribble.annotation;
 
 /**
- * Enum for strand, which can be encoded as string
+ * Enum for strand, which can be encoded as a string
  */
 public enum Strand {
-    POSITIVE("+"), NEGATIVE("-"), NONE("!");  // not really sure what we should do for the NONE Enum
 
     /**
-     * How we represent the strand information as text
+     * Represent the positive or forward strand.
      */
-    private String encoding;
-    Strand(String str) {
-        encoding = str;
+    POSITIVE('+'),
+    /**
+     * Represent the negative or reverse strand.
+     */
+    NEGATIVE('-'),
+
+    /**
+     * Denotes that an strand designation is not applicable
+     * or is unknown.
+     */
+    NONE('!');  // not really sure what we should do for the NONE Enum
```

Comment:
Now that this is refactored, maybe is a good time to add two instances for no strand to distinguish between non-available (not parsed, column not present in the file) and no strand (`.` in the BED format).

Reply:
I'm happy to change '!' for a '.' ... in what situation/format a '!' would mean no-strand? 

However I'm not sure about adding more NONEs... I would like to see a situation in where one would need to distinguish between more than one NONE.

In any case one could use ```Optional<Strand>``` to add an extra NONE... or even drop the current one!